### DISCUSSION TITLE
Improving funding process Algorand

### DISCUSSION THREAD

**Discussion by ipaleka** [asastats/channel#242] [General]
Here's a conversation initiated by Lucas from summitto, they are building a new system for receiving grants from Algorand Foundation.

- - - - -
Dear Ivica,

We developed a new grants system on behalf of the Algorand Foundation to improve the funding process of the Algorand ecosystem. This tool can help to increase your funding.

We are currently gathering feedback for the Foundation by asking the ecosystem before they  launch and would love to learn what you think about our tool. Would you be open for a video call and otherwise to fill out a short survey form? You can already find the website on testnet here: https://matchfund.app/

Let me know what you think.

Kind regards,

Lucas Mul
- - - - -

Hi Lucas!

We are indeed interested in an improvement of the Algorand Foundation grant system as we are one of the rare stable projects in the ecosystem that hasn't received any funds so far.

Btw, everything in ASA Stats is transparent, so I'd stream this conversation to our official communication channel on GitHub if/after you confirm and allow that.

Would you insist that I personally attend that video call or the invitation is valid for some other (prominent) member of our community too?

Sincerely,

Ivica
- - - - -
Thanks for the kind and quick response! That's an interesting case, so your feedback would be especially helpful.

As for sharing information on Github, we're happy if you share summaries of what we discuss but as we're still in an early feedback phase, we prefer not to share entire transcripts/video recordings.

Regarding who should attend the video call: any feedback from your community would be extremely valuable, so if someone else will attend, that would also be fine! 
- - - - -
I was referring to this conversation, I want to stream it in a discussion on our GitHub as we do that for pretty much every non-confidential material.

And about the video call, I'm completely focused on development so I'd rather that two or three of our community members attend instead of me. They would probably be among those who will prepare our application through your app once when you deploy it, I probably won't have time for anything related.
- - - - -
Then I misunderstood. Sorry about that. Of course, no problem with streaming this conversation to Github!

Sounds good. If you could bring me in contact with them, that would be great. Thank you!
- - - - -
Thank you, we're going to organize a discussion, and afterward I'll reach you with a few of our prominent members contacts.
- - - - -
Perfect, looking forward to it!
- - - - -
At least two of our members are ready to attend the meeting, I want to check with you if that's an optimal number of attendees.

What will be the next step, do you need their emails or you'll send an invite that I should redirect?
- - - - -
That would be great!

Regarding the next step, I'm still quite flexible for next week, so if you could check if they have any availability next week that might be the easiest. Then I can send out an invite. If you prefer me to arrange a date with your member myself, then that would also be fine! Then I would indeed need their emails.
- - - - -

So we'll need at least two people who can attend a video call. I guess we're looking for both people who can test the app and for those who will take part in our actual grant request on their platform once it is deployed and active.

**Comment by ipaleka** [comment]
ipaleka — Today at 4:02 PM in [Discord](https://discord.com/channels/906917846754418770/914540797054025788/1021783478414618674):

> So we'll need at least two people who can attend a video call. I guess we're looking for both people who can test the app and for those who will take part in our actual grant request on their platform once it is deployed and active.

Obvious choices are @Damo and @AlgoRhythMatic ... @SCN9A and @kerrilija haven't attended any meeting, so they are prospects too. As always, this is also a call for all DAO governors. 
- - - - -
kerrilija — Today at 8:14 PM

I agree with you. Anyway, if @AlgoRhythMatic or @Damo are unable to participate, I will most likely be available and willing to help with this. Nevertheless, I would like to be up to date with the process and be more prepared for future events such as this.
- - - - -
ipaleka — Today at 8:35 PM

There are no obstacles that the three of you attend.
Regarding the process, once the other party allows public communication ("Of course, no problem with streaming this conversation to Github!") I create a discussion and everybody's informed for the first time. After that we're collecting attendees in the discussion and when we're ready we inform the other party about the attendees. From that moment other party and attendees pick the date for a call.
After the meeting is over, one or more of our attendees create a report and it is posted in GitHub. For every meeting there's a budget for attending the meeting and for report creation.

**Comment by ipaleka** [comment]
ipaleka — Today at 9:48 PM in [Discord](https://discord.com/channels/906917846754418770/914540797054025788/1021870511464128543):

Just a reminder that this grants system improvement is a new opportunity for us. If someone has missed, they refused us last time https://discord.com/channels/@me/1003566984006479879/1021869346907246642 as they already funded a similar project.
This time we can go with several subprojects, if not dozens, some of those are:
- open-sourcing the ASA Stats codebase
- development of ASA Stats NFT prices evaluation by using AI 
- developing and audit of the ASA Stats Permission ASC
- funding of our user widgets system
- funding particular user-widgets development and audit 
- - - - -
ipaleka — Today at 9:54 PM

Btw, @DragMZ I really think your terminal app is a beautiful and solid prospect for a grant.
- - - - -
DragMZ — Today at 9:57 PM

I'm still not sure what the terminal app is. Actually it's an experiment to see if I can use it for trading.
- - - - -
ipaleka — Today at 9:58 PM

Well, whatever it is I think they'll like it enough to provide funds for its development! 🙂

**Reply by AlgoRhythMatic** [reply]
> I'm still not sure what the terminal app is. Actually it's an experiment to see if I can use it for trading.

IMO, this statement indicates your example is both the purest + first "real life" API use case we've come across yet.  You need the API for a specific purpose (or at least think you might), and are in turn creating an application to leverage it for your personal benefit.  We need to record it, talk about it, and use it as prime example as to how others may do the same - regardless of whether they are a single 1-man team, or an enterprise-level organization.

**Reply by AlgoRhythMatic** [reply]
> Obvious choices are @damo and @AlgoRhythMatic ... @SCN9A and @kerrilija haven't attended any meeting, so they are prospects too. As always, this is also a call for all DAO governors.

I would be happy to participate, but depending on date - I may be on vacation and away from the digital world (starting a few days from now).

**Reply by ipaleka** [reply]
> I would be happy to participate, but depending on date - I may be on vacation and away from the digital world (starting a few days from now).

Enjoy your vacation :), we'll manage this!

**Reply by wxivd** [reply]
@ipaleka @AlgoRhythMatic Thanks for the opportunity.

- I will make time to test the tool and construct a feedback. 

- Attending the meeting on Tuesdays / Saturdays would be ideal for me between 10AM-3PM GMT, as I am off-work on these days. I am keen to know how this fits the schedule of the attendees. 

I am excited to meet new teams and looking forward to it. Given that this is the first meeting for me to attend it would ideal to get a sort of meeting goals from both sides perspectives.

**Reply by ipaleka** [reply]
I'm so glad you are available. As of now, you and Damo are ready to attend, we'll see with @kerrilija and with Lucas about the optimal number of participants.

**Comment by wxivd** [comment]
Meeting Notes: Algorand quadratic funding matchfund

### **Meeting attendees**
@Damolicious  and @SCN9A met with Lucas from summitto on 27 September 2022, 12pm - 12:30pm GMT

### **Meeting Objectives**
They developed a new grants system on behalf of the Algorand Foundation to improve the funding process of the Algorand ecosystem. They are gathering feedback for the Foundation by asking the ecosystem before they launch. 

In this meeting ASA Stats provided a feedback, and raised some questions. 

### **A: Feedback and suggestions by ASA Stats**
The list is meant to convey the key points 

1. [Landing Page](https://matchfund.app/): 
   - Simple design, animated logos of various projects gives the symbolic impression of thriving projects
   - Key messages may be more focused on
     - Purpose: "Support your favourite Algorand projects" rather than "Support your interests"
     - layperson (Community): "Leverage the power of quadratic funding" further explained

2. Sign up
   - Registration process requires: first, last name, email address, password, password conformation. 
   - Add sign-up security code sent to email (Authentication of email access)
   - Remove "Sign up with email" from landing page following registration and log-in.

3. Profile Settings
   - Update profile (first name, last name, website, location, Bio) is not functional "Failed to update profile"
   - Add ability to change email, password

4. [Projects](https://matchfund.app/#projects) ([Explored](https://matchfund.app/projects/6))
   - Donate function allows two types of donations: Algorand Standard Asset and Algorand Coin
    i- Algorand Coin: an interface showing a predefined range of Algo donations (5A, 10A...100A) with a custom field for user-provided amount. The metric "Match you generate" is calculated before actual Algo donation.
    ii- Algorand Standard Asset: required wallet log-in to allow selection of ASA opted-in. **We highlighted ASA Stats engine to calculate real value of ASA donated.** 

5. Checkout page
   - Value of Match-generated is rounded. (In donation page: 100A donation = 89.45 Match generated), on checkout that Match generated is 90. Suggested unity in value presented.
   - Selected project to donate to is removed after sign out (Firefox). 
   - Add ability to select more than one project for donations; Upon adding a new project, it replaces previous selection.
   - Support other wallet options.

6. Create New Project: entails three steps
i. Project details: Project name, Algo wallet address, homepage, Github page, short description, Country, Profile pic.
ii. Describe your project: 5 Sections: About project, Use of funds, Timeline, About company, Project screenshot.
iii. Review and submit.

   - Add save progress functionality. Sign-out results in loss of information provided, clicking on back removes inserted content. 
   - Get profiled link / downloadable file for projects to plan it out
   - After submission "Your project has been submitted!": add information to set expectations (What is next, Who is reviewing, Time-expectations).
   - Add my projects to user setting with status (Pending review, published..etc) 
   - Add account functionality such as saving previous application, re-publish with edits.
   - Consider projects' Rating criteria based on the fulfilment of development objectives in previous round of raising funds with consideration to time. It shows commitment and facilitate further funding
   - Perks to donators functionality, allow projects the ability to reward donators 

### **Questions from a project perspective**
- **Funding threshold**, the minimum amount that must be reached in order to successfully complete a funding. on Matchfund, the project description contains 5 items (of relevance here are Use of Funds & Timeline).
  - What will happen if total amount of funds raised did not reach the funding threshold for proposed development? consequences would place much strain on projects to fulfil objectives with funding shortage. 
  - Given the nature of projects in development, projects applying for funding may target a set of features to be developed by project team, the number of which (Items / fund utilisation) depends of the amount of generated funding. The point here is that fund utilisation may Better be tied to deliverables items.

### **Questions from a community/user perspective**
- How would Matchfund authenticate published projects? Make sure they are who they claim to be?
- Given that quadratic funding is mathematically optimised fund-matching where emphasis is placed on the number of contributors (promoting democracy in funding decisions) rather than the amount funded; limiting the impact of centralised decisions (whales, corporate funds). How would Matchfund prevent bad-practice such as a single entity donating from multiple accounts?
- Amount of Algo for funding round is fixed, this raises two questions: 
  1. How does the match generated change across the funding round period. (Start/mid/end)?
  2. The metric "match you generated for this project" should be changed to an **estimate** 
- How can a keen community member know the rounds schedule?


Overall, it was pleasant meeting by all means, Lucas would convey those points to the foundation to get some answers.
